python/ray/rllib/examples/multiagent_cartpole.py

```diff
@@ -15,6 +15,7 @@
 import argparse
 import gym
 import random
+import distutils.version
```

Comment:
I think the examples which are public facing don't need to support older versions of TF.

Reply:
done